blog.html

```diff
@@ -22,6 +22,146 @@
+                    <li>`withRange(null, upper)` == `withUpperBound(upper)`</li>
+                    </ul>
+                </li>
+                </ul>
+                <h3>Kafka Connect</h3>
+                <ul>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-793%3A+Allow+sink+connectors+to+be+used+with+topic-mutating+SMTs">KIP-793</a>: Allow sink connectors to be used with topic-mutating SMTs: </b><br>Adds support for topic-mutating SMTs for async sink connectors. This is to address an incompatibility between sink connectors overriding the SinkTask::preCommit method and SMTs that mutate the topic field of a SinkRecord .</li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-875%3A+First-class+offsets+support+in+Kafka+Connect">KIP-875</a>: First-class offsets support in Kafka Connect: </b><br>Provides first-class admin support for offsets in Kafka Connect. KIP-875 Part 1 added endpoints to get offsets and a new STOPPED state for connectors. The alter offsets and reset offsets endpoints have now been added.
+                        <table>
+                            <tr>
+                                <th>Action</th>
+                                <th>Description</th>
+                            </tr>
+                            <tr>
+                                <td>GET /connectors/{connector}/offsets</td>
+                                <td>Retrieve the offsets for a connector; the connector must exist</td>
+                            </tr>
+                            <tr>
+                                <td>PATCH /connectors/{connector}/offsets</td>
+                                <td>Alter the offsets for a connector; the connector must exist, and must be in the STOPPED state
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>DELETE /connectors/{connector}/offsets</td>
+                                <td>Reset the offsets for a connector; the connector must exist, and must be in the STOPPED state
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>PUT /connectors/{connector}/pause</td>
+                                <td>Pause the connector; the connector must exist</td>
+                            </tr>
+                        </table>
+                    </li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-898%3A+Modernize+Connect+plugin+discovery">KIP-898</a>: Modernize Connect plugin discovery: </b><br>With KIP-898, Connect workers now read from ServiceLoader manifests and module info directly during startup for more efficient plugin class discovery. Note that this update requires connector developers to add service declarations to their plugins.</li>
```

Comment:
```suggestion
                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-898%3A+Modernize+Connect+plugin+discovery">KIP-898</a>: Modernize Connect plugin discovery: </b><br>With KIP-898, Connect workers can now read from ServiceLoader manifests and module info directly during startup for more efficient plugin class discovery. See the <a href="https://kafka.apache.org/36/documentation.html#connect_plugindiscovery">migration documentation</a> for how to add service declarations to plugins to opt-in to these optimizations.</li>
```

Reply:
Alternatively, just soften "requires" so that someone reading the release blog and not the KIP doesn't get the impression that this is a breaking change.

Reply:
instead of "requires", added "allows".  With that, it becomes `Note that this update allows connector developers to add service declarations to their plugins.`